portal-api/src/v1/gef/controllers/externalApi.controller.js

```diff
@@ -107,11 +107,9 @@ exports.getAddressesByPostcode = async (req, res) => {
       ]);
     }
 
-    const response = await externalApi.ordnanceSurvey.getAddressesByPostcode(postcode);
+    const response = await externalApi.geospatialAddresses.getAddressesByPostcode(postcode);
 
-    const addresses = [];
-
-    if (!response?.data?.results) {
+    if (response.status !== 200) {
```

Comment:
* Http Status code on the file please.
* Please add an explanation why we are returning `422` when the response is not `200` for clarity thanks.

Reply:
422 is quite commong code for DTFS errors. I left the same code.
I guess it means that client sent Bad Request, but error happened in the backend.